hcbedfile/README.txt

```diff
@@ -0,0 +1,36 @@
+\scripts
+	[generate_sambamba.py]
+	[make_hc_scripts.py]
+	[make_hcbedfile_part1.sh]
+	[make_hcbedfile_part2.sh]
+	[make_population_sample_list.py]
+
+
+\generate_sambamba.py
+	Script that generates sambamba jobs for a set of BAM files.
```

Comment:
Sambamba is a collection of tools. This minimal description does not explain what the script does